frontend/src/metabase/dashboard/components/DashCard/DashCardVisualization.tsx

```diff
@@ -224,6 +225,18 @@ function DashCardVisualization({
     parameterValuesBySlug,
   ]);
 
+  const inIframe = () => {
+    try {
+      return window.self !== window.top;
+    } catch (e) {
+      return true;
+    }
+  }
+
+  if (inIframe() !== true && isEmbed === true) {
```

Comment:
Not a tsx expert but seems like this could be

```Typescript
if (!inIframe() && isEmbed) {
    ...
}
```

Reply:
Fixed @rafaelvrd